src/dom/attributes/attribute-manager.js

```diff
@@ -9,6 +9,55 @@
  *
  * The Attribute Manager is specific to adding custom attributes and describing their implementation (handlers). (The [Dom Manager](../../) contains the general implementation.)
  *
+ *
+ * **Examples**
+ *
+ * Built-in attribute managers like `data-f-value` and `data-f-foreach` automatically bind variables in your project's model to particular HTML elements. However, the value of the variable itself isn't always what you want in the UI.
+ *
+ * One application of custom attribute managers is to display particular HTML elements or content based on the value of a model variable, without displaying the value of the variable itself. For example, when your model variable meets a certain condition, you may want to show specific text in your UI based on the value of that variable. Creating a custom attribute manager is an easy way to do this:
+ *
+ *      Flow.dom.attributes.register('showProfitFeedback', '*', function(profit) {
+ *            var profitStr = '';
+ *
+ *            if (profit > 0.10) {
+ *                  profitStr += 'Great work so far! Your firm is doing extremely well.';
+ *            } else if (profit > 0) {
+ *                  profitStr += 'Your firm is profitable, but there is room for improvement.';
+ *            } else {
+ *                  profitStr += 'Careful, your firm is currently running at a loss. Consider strategies to increase sales or decrease spending.'
```

Comment:
nitpick, but lets just do `message` instead of `profitStr` and `=` instead of `+=`